releaseNotes/SpringCloudAzureAppConfigurationConfig.md

```diff
@@ -6,6 +6,14 @@
 
 [Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
 
+## 1.1.4/1.2.4 - April 27, 2020
+
+* Add ability to modify client connection to App Configuration and Key Vault. Clients can be provided via ```ConfigurationClientBuilderSetup``` and ```SecretClientBuilderSetup```.
```

Comment:
Suggestion : Added support to allow users to configure the client used to connect to App Configuration and Key Vault